docs/CONTRIBUTING.md

```diff
@@ -41,44 +42,52 @@ Be sure to mention that you're using Connectors and also let us know what servic
 You can also find us in the `#enterprise-search` channel of the [Elastic Community Slack](http://elasticstack.slack.com).
 
 ## Types of contributions
-### Initial contribution
+
+### Customize a connector
+
+To customize an _existing_ connector, follow these steps:
+
+1. Customize the source file for your data source from [connectors/sources](../connectors/sources)
+2. Add a unit test in [connectors/sources/tests](../connectors/sources/tests) with **+90% coverage**
+3. Declare your dependencies in [requirements.txt](../requirements/framework.txt). Make sure you pin these dependencies.
+4. For each dependency you add (including indirect dependencies) list all licences and provide the list in your patch.
+5. When possible, provide a docker image that runs the backend service, so we can test the connector. If you can't provide a docker image, provide the credentials needed to run against an online service.
```

Comment:
This would be important for a net-new connector, but I'm not sure its necessary for existing data source types.